Expensify/App
# Add a Settings page to configure people in workspace

### Details

This PR adds a workspace configuration page where admins can remove people from their workspaces or use modal to invite new members.

cc @marcaaron @MitchExpensify @Jag96 @madmax330 

### Fixed Issues

Fixes https://github.com/Expensify/Expensify/issues/166468

### Tests
1. Create a free workspace policy via the global create menu
1. Navigate to the settings screen and verify this workspace is visible
1. Tap the workspace
1. Tap the "People" option on the next screen
1. If there is only you, the admin, in the workspace, use the green `Invite` button which will bring in a modal from a side. Enter an email of an account you want to add to the workspace.
1. Once you added the number of accounts to the workspace you would wish, select some using the checkboxes on the left side. As soon as you select one checkbox, the `Remove` button should turn bright red. Click on the `Remove` button.
1. A confirmation modal looking like this should show up. Click `Remove` again to confirm the removal.
![image](https://user-images.githubusercontent.com/36083550/123350880-ce97f880-d553-11eb-85e4-cf9c8b8e93bf.png)
1. Make sure the person selected has been removed from the list and it does not show up even after page refresh.
1. Now, try selecting the first uppermost checkbox and make sure that all the checkboxes has been checked.
1. Go on and try to remove all of the members of the workspace.
1. Your account, the admin, should stay unremoved after this action. That is the correct behaviour, you cannot remove admin of workspace through this page.

### QA Steps


### Tested On


### Screenshots


#### Web
![2021-06-30_10-43-36](https://user-images.githubusercontent.com/32969087/124028740-107bd100-d990-11eb-8edf-db478fa31ae8.png)

#### Mobile Web
![2021-06-30_10-48-56](https://user-images.githubusercontent.com/32969087/124029497-eaa2fc00-d990-11eb-8cd8-f431d8a5ace3.png)

#### Desktop
![2021-06-30_10-53-24](https://user-images.githubusercontent.com/32969087/124029985-6ac96180-d991-11eb-97b7-321bf72befa7.png)

#### iOS
![2021-06-30_10-41-48](https://user-images.githubusercontent.com/32969087/124028754-140f5800-d990-11eb-936c-17cd53aa0105.png)

#### Android
![2021-06-30_10-55-46](https://user-images.githubusercontent.com/32969087/124030273-bd0a8280-d991-11eb-9257-91a4e606f71e.png)
